web/views.py

```
@@ -7054,16 +7502,20 @@ def get_client_secret(subscription):
 
     except json.JSONDecodeError as e:
-        return JsonResponse({"error": f"Invalid JSON: {str(e)}"}, status=400)
+        logger.warning("Invalid JSON in create_membership_subscription: %s", str(e))
+        return JsonResponse({"error": "Invalid JSON format"}, status=400)
     except stripe.error.CardError as e:
-        return JsonResponse({"error": f"Card error: {str(e)}"}, status=400)
+        logger.warning("Card error in create_membership_subscription: %s", str(e))
+        return JsonResponse({"error": "Card payment failed"}, status=400)
     except stripe.error.StripeError as e:
-        return JsonResponse({"error": f"Payment processing error: {str(e)}"}, status=500)
+        logger.error("Stripe error in create_membership_subscription: %s", str(e))
```

Ruff (0.11.9)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)